src/libraries/System.Security.Cryptography/src/Resources/Strings.resx

```diff
@@ -834,6 +834,9 @@
   <data name="PlatformNotSupported_CryptographyOpenSSLNotFound" xml:space="preserve">
     <value>OpenSSL is required for algorithm '{0}' but could not be found or loaded.</value>
   </data>
+  <data name="PlatformNotSupported_CryptographyOpenSSL3NotFound" xml:space="preserve">
+    <value>OpenSSL 3.0 or higher is required but could not be found or loaded.</value>
+  </data>
```

Comment:
The parallel one for ENGINE uses feature description, not a version number.  I recommend doing the same here.

```suggestion
  <data name="PlatformNotSupported_CryptographyOpenSSLProvidersNotSupported" xml:space="preserve">
    <value>OpenSSL providers are not supported on this platform.</value>
  </data>
```